videos/videos_router.js

```diff
@@ -13,6 +13,15 @@ router.get("/", async (req, res) => {
   }
 });
 
+router.get("/random9", async (req, res) => {
+  try {
+    const {rows} = await Videos.find9();
+    res.status(200).json({rows});
+  } catch ({message}) {
+    res.status(500).json({errorMessage: `Encountered |*||${message}|*| while retrieving videos from the database.`});
```

Comment:
What would this message be? Should this be `err`?

Reply:
Err would be more declarative, wouldn't it.
 Sounds good.

Reply:
updates response variable at this commit  https://github.com/Lambda-School-Labs/artificial-artist-be/pull/23/commits/e3b43a185cc003abbb56df91647f044b687e5062